app/src/main/java/org/literacyapp/authentication/AuthenticationActivity.java

```diff
@@ -117,6 +127,10 @@ public Mat onCameraFrame(CameraBridgeViewBase.CvCameraViewFrame inputFrame) {
         // Add overlay
         animalOverlayHelper.addOverlay(imgRgba);
 
+        if (faceDetected && !isFaceInsideFrame){
+            DetectionHelper.drawArrowFromFaceToFrame(animalOverlay, imgRgba, face);
```

Comment:
doesn't look really nice but should work for a start (maybe together with audio?)